Maestro/subscriptions.json

```diff
@@ -854,6 +854,8 @@
         "https://github.com/dotnet/sdk/blob/rel/**/*",
         "https://github.com/dotnet/sdk/blob/release/**/*",
         "https://github.com/dotnet/dotnet-cli-archiver/blob/master/**/*",
+        "https://github.com/dotnet/cli-migrate/blob/master/**/*",
+        "https://github.com/dotnet/clicommandlineparser/blob/master/**/*",       
```

Comment:
Do you know if these have Maestro webhooks set up for them? (A repo admin needs to set this up before first use.)

Instructions: https://github.com/dotnet/core-eng/blob/master/Documentation/Maestro/web-hooks.md

Reply:
Yes. I added those already.